WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressFormViewController.swift

```diff
@@ -615,6 +621,10 @@ private extension ShippingLabelAddressFormViewController {
                                                             comment: "Error in finding the address in the Shipping Label Address Validation in Apple Maps")
         static let phoneNumberErrorNotice = NSLocalizedString("The phone number is not valid or you can't call the customer from this device.",
             comment: "Error in calling the phone number of the customer in the Shipping Label Address Validation")
+        static let addressValidationErrorNotice = NSLocalizedString("Certain required fields needs attention.",
+                comment: "Ship From address update failed notice")
```

Comment:
We can probably clarify this comment to help translators understand better:
```suggestion
                comment: "Error message when local validation fails in Shipping Label Address Validation")
```